src/container/container_iv.c

```diff
@@ -195,7 +195,7 @@ cont_iv_ent_copy(struct ds_iv_entry *entry, struct cont_iv_key *key,
 		dst->iv_agg_eph.eph = src->iv_agg_eph.eph;
 		break;
 	default:
-		D_ERROR("bad iv_class_id %d: "DF_RC".\n", entry->iv_class->iv_class_id,
+		D_ERROR("bad iv_class_id %d: "DF_RC"\n", entry->iv_class->iv_class_id,
```

Comment:
This is failing the clang-format check:
https://github.com/daos-stack/daos/actions/runs/6647367242

Reply:
Better to use `DL_ERROR` here and then the warning won't apply anyway.
```suggestion
		DL_ERROR(-DER_INVAL, "bad iv_class_id %d, entry->iv_class->iv_class_id);
```